cmd/web_letsencrypt.go

```diff
@@ -35,7 +38,26 @@ func runLetsEncrypt(listenAddr, domain, directory, email string, m http.Handler)
 
 	magic := certmagic.NewDefault()
 	magic.Storage = &certmagic.FileStorage{Path: directory}
+	// Try to use private CA root if provided, otherwise defaults to system's trust
+	var CertPool *x509.CertPool = nil
```

Comment:
```suggestion
	var certPool *x509.CertPool = nil
```

The local variable name should be `camelCase`